server/controller/get_subscriptions.go

```diff
@@ -0,0 +1,36 @@
+package controller
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-confluence/server/service"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+var autocompleteGetChannelSubscriptions = &Endpoint{
+	RequiresAdmin: true,
+	Path:          "/autocomplete/GetChannelSubscriptions",
+	Method:        http.MethodGet,
+	Execute:       handleGetChannelSubscriptions,
+}
+
+func handleGetChannelSubscriptions(w http.ResponseWriter, r *http.Request) {
+	channelID := r.FormValue("channel_id")
```

Comment:
@iomodo, I'm not sure of the exact commit needed from master, but I believe this is the functionality that is not currently in a released version?

Reply:
There was a change to autocomplete that now passes the `channel_id` to the URL as a query param.  This was only recently introduced.  Without this new feature, you have no way of detecting the current channel for which you are typing an autocomplete command.

Reply:
True, changes are for 5.26 so you can grab the lates master for now.

Reply:
@jfrerich @iomodo Do we need to update the minimum server version because of this change?

Reply:
Yes, we needed to update the min server version. I committed the `min_server_version` to `5.26.0`.  

What is the correct action with respect to merging? I added the `Do Not Merge/Awaiting PR` label.  We can merge post `5.26.0` release.  How does that sound and is that necessary?

Reply:
@iomodo, @mickmister 

* updated `min_server_version` to `v5.26.0` in plugin.json
* updated go.mod with latest server version commit via `go get github.com/mattermost/mattermost-server/v5@93a537a636299e1e049505bdaf072f8d3ae974a4`